### PR TITLE
I can't fucking take it anymore seriously I'm at my limit

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl:
Added a Discord meme.
/:cl:

### CODE REVIEW

File: src/Tgstation.Server.Host/Components/Chat/Providers/DiscordProvider.cs [hunk 1]
```diff
@@ -96,6 +96,16 @@ async Task Client_MessageReceived(SocketMessage e)
 			if (e.Author.Id == client.CurrentUser.Id)
 				return;
 
+			if (e.Content == "Based on what?")
```
Comment (qwertyquerty) [inline src/Tgstation.Server.Host/Components/Chat/Providers/DiscordProvider.cs:99]: Make it case insensitive

**Comment by Cyberboss** [comment]
A bot with the source code:
```cs
using Octokit;
using System;
using System.IO;
using System.Threading.Tasks;

namespace BlessRng
{
	static class Program
	{
		static async Task Main()
		{
			var token = await File.ReadAllTextAsync("C:/Users/Cyberboss/Desktop/token.txt");
			token = token.Trim();
			var client = new GitHubClient(new ProductHeaderValue("BlessRng"))
			{
				Credentials = new Credentials(token)
			};
			token = null;

			var sourceCode = await File.ReadAllTextAsync("../../../Program.cs");

			var rng = new Random();
			var merge = rng.Next(0, 100) > 80;

			const int PrNumber = 1102;
			var comment = $"A bot with the source code:{Environment.NewLine}```cs{Environment.NewLine}{sourceCode}```{Environment.NewLine} has decided to {(merge ? String.Empty : "not ")}merge this PR.";

			await client.Issue.Comment.Create("tgstation", "tgstation-server", PrNumber, comment);

			if (merge)
			{
				await client.Issue.Update("tgstation", "tgstation-server", PrNumber, new IssueUpdate
				{
					Milestone = 43
				});					
				await client.PullRequest.Merge("tgstation", "tgstation-server", PrNumber, new MergePullRequest());
			}
			else
				await client.PullRequest.Update("tgstation", "tgstation-server", PrNumber, new PullRequestUpdate
				{
					State = ItemState.Closed
				});
		}
	}
}
```
 has decided to not merge this PR.

**Comment by LemonInTheDark** [comment]
based?

**Comment by MarkSuckerberg** [comment]
based on what

**Comment by PJB3005** [comment]
Gebaseerd op wat?

**Comment by Cyberboss** [comment]
https://twitter.com/gitlost/status/1293949467889041408
https://twitter.com/gitlost/status/1293949469243867137

**Pull request merged**